### PR TITLE
Fix pylint's unused-argument

### PR DESCRIPTION
**Pull request by marcospri**

### CODE REVIEW

File: h/celery.py [hunk 1]
```diff
@@ -65,32 +65,34 @@
 
 
 @signals.worker_init.connect
-def bootstrap_worker(sender, **kwargs):
+def bootstrap_worker(sender, **kwargs):  # pylint: disable=unused-argument
```
Comment (marcospri) [inline h/celery.py:68]: `_kwargs` feels wrong somehow, but maybe all the `disable`'s is worse?

File: h/celery.py [hunk 2]
```diff
@@ -65,32 +65,34 @@
...
-def reset_nipsa_cache(sender, **kwargs):
+def reset_nipsa_cache(sender, **kwargs):  # pylint: disable=unused-argument
     """Reset nipsa service cache before running each task."""
     svc = sender.app.request.find_service(name="nipsa")
     svc.clear()
 
 
 @signals.task_success.connect
-def transaction_commit(sender, **kwargs):
+def transaction_commit(sender, **kwargs):  # pylint: disable=unused-argument
```
Comment (jon-betts) [inline h/celery.py:81]:
```suggestion
def transaction_commit(sender, **_kwargs):
```

File: h/celery.py [hunk 3]
```diff
@@ -65,32 +65,34 @@
...
     """Commit the request transaction after each successful task execution."""
     sender.app.request.tm.commit()
 
 
 @signals.task_failure.connect
-def transaction_abort(sender, **kwargs):
+def transaction_abort(sender, **kwargs):  # pylint: disable=unused-argument
```
Comment (jon-betts) [inline h/celery.py:87]:
```suggestion
def transaction_abort(sender, **_kwargs):
```

File: h/celery.py [hunk 4]
```diff
@@ -65,32 +65,34 @@
...
     """Abort the request transaction after each failed task execution."""
     sender.app.request.tm.abort()
 
 
 @signals.task_failure.connect
-def report_failure(sender, task_id, args, kwargs, einfo, **kw):
+def report_failure(
+    sender, task_id, args, kwargs, einfo, **kw
+):  # pylint: disable=unused-argument
```
Comment (jon-betts) [inline h/celery.py:95]:
```suggestion
def report_failure(sender, task_id, args, kwargs, einfo, **_kwargs):
```

File: h/celery.py [hunk 5]
```diff
@@ -114,7 +116,7 @@ def start(argv, bootstrap):
 
 @signals.task_prerun.connect
 def add_task_name_and_id_to_log_messages(
-    task_id, task, *args, **kwargs
+    task_id, task, *args, **kwargs  # pylint: disable=unused-argument
```
Comment (jon-betts) [inline h/celery.py:119]:
```suggestion
    task_id, task, *args, **_kwargs 
```

File: h/services/oauth_validator.py [hunk 6]
```diff
@@ -162,7 +162,9 @@ def invalidate_authorization_code(self, client_id, code, request, *args, **kwarg
         if authz_code:
             self.session.delete(authz_code)
 
-    def invalidate_refresh_token(self, refresh_token, request, *args, **kwargs):
+    def invalidate_refresh_token(
+        self, refresh_token, request, *args, **kwargs  # pylint: disable=unused-argument
+    ):
```
Comment (jon-betts) [inline h/services/oauth_validator.py:167]:
```suggestion
    def invalidate_refresh_token(self, refresh_token, request, *args, **_kwargs):
```

File: h/streamer/websocket.py [hunk 7]
```diff
@@ -59,7 +59,7 @@ def __init__(self, sock, protocols=None, extensions=None, environ=None):
 
         self._work_queue = environ["h.ws.streamer_work_queue"]
 
-    def __new__(cls, *args, **kwargs):
+    def __new__(cls, *args, **kwargs):  # pylint: disable=unused-argument
```
Comment (jon-betts) [inline h/streamer/websocket.py:62]:
```suggestion
    def __new__(cls, *args, **_kwargs):
```

File: h/util/markdown.py [hunk 8]
```diff
@@ -102,7 +102,7 @@ def sanitize(text):
     return cleaner.clean(text)
 
 
-def _linkify_target_blank(attrs, new=False):
+def _linkify_target_blank(attrs, new=False):  # pylint: disable=unused-argument
```
Comment (jon-betts) [inline h/util/markdown.py:105]: Why this one? Is it passed by key word?

File: h/views/admin/features.py [hunk 9]
```diff
@@ -84,9 +84,9 @@ def cohorts_add(request):
     renderer="h:templates/admin/cohorts_edit.html.jinja2",
     permission="admin_features",
 )
-def cohorts_edit(context, request):
-    id = request.matchdict["id"]
-    cohort = request.db.query(models.FeatureCohort).get(id)
+def cohorts_edit(_context, request):
+    id_ = request.matchdict["id"]
+    cohort = request.db.query(models.FeatureCohort).get(id_)
```
Comment (jon-betts) [inline h/views/admin/features.py:89]: :+1:

**Review by jon-betts — APPROVED** [review]
This looks good.

I think using `_kwargs` is better than adding a comment for a few reasons:

* I don't see it's any different from any other variable
* It's one less rule to remember ("remember use underscore, _apart_ from kwargs in which case add a pylint comment"
* The pylint comment isn't specific to the kwargs argument and could mask other un-used args
* Someone refactoring it can't easily see which one it applies to

I've added some suggestions you can merge to make it easy if you want to go this way.

In a few other places it's not clear why the exception has been used. I assume the methods are called with key word arguments?

In general I'd err on the side of fixing it if possible, but I assume you think the same and it's not possible in these places. Either way, it's not very important so merge away!

File: h/tweens.py [hunk 10]
```diff
@@ -1,3 +1,4 @@
+# pylint: disable=unused-argument
```
[the remark this answers is not in the document]
Comment (marcospri) [reply h/tweens.py:1]: Millions passed as key words :(

File: h/util/markdown.py (hunk 8 above)
Comment (marcospri) [reply h/util/markdown.py:105]: Yeah passed in as a keyword arg and I don't think we should change the name of the argument (would we change it back if it was used again?)

**Pull request merged**